docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -151,42 +151,24 @@ Teleport will respond back with a discovery info:
 ```bash
 $ curl https://tele.example.com/webapi/ping
 
-{"server_version":"5.0.0-dev","min_client_version":"3.0.0"}
+{"server_version":"6.0.0","min_client_version":"3.0.0"}
 ```
 
 ## Step 2/3 Create a local admin
 
 Local users are a reliable fallback for cases when the SSO provider is down.
-Let's create a local admin `alice` who has access to Kubernetes group `system:masters`.
-
-Save this role as `member.yaml`:
-
-```yaml
-kind: role
-version: v3
-metadata:
-  name: member
-spec:
-  allow:
-    # This field is used for SSH logins. You have to keep 'logins' as a non-empty random value
-    # for Kubernetes to work until we fix it.
-    logins: ['keep-this-value-here']
-    kubernetes_groups: ["system:masters"]
-```
+Let's create a local admin `teleport-admin` who has access to Kubernetes group `system:masters`.
 
 Create the role and add a user:
 
 ```bash
 # To create a local user, we are going to run Teleport's admin tool tctl from the pod.
 $ POD=$(kubectl get po -l app=teleport-cluster -o jsonpath='{.items[0].metadata.name}')
 
-# Create a role
-$ kubectl exec -i ${POD?} tctl -- create -f < member.yaml
-
 # Generate an invite link for the user.
-$ kubectl exec -ti ${POD?} tctl -- users add alice --roles=member
+$ kubectl exec -ti ${POD?} -- tctl users add teleport-admin --roles=editor,access --kubernetes_groups=system:masters --logins=k8s-keep-this-login
```

Comment:
I've added `--kubernetes_groups=system:masters --logins=k8s-keep-this-login` since the new default roles don't have anything set. If there is role with less privilege you think I should use please let me know.

Reply:
Why that change? The role I had above was a bit simpler I think

Reply:
I've changed it back to how it was before.